css/highlight.css

```diff
@@ -1,4 +1,20 @@
-.highlight {line-height:20px;  padding:10px 10px 10px 10px; margin: 20px; margin-bottom: 10px; display: block;  font-size:14px;  font-family: Monaco, Consolas, Courier, monospace;  letter-spacing:0.4px;  -webkit-border-radius: 10px;  -moz-border-radius: 10px;background-color: black; font-style: normal }
+pre {
+    background-color: black;
```

Comment:
this is the most important line. since it was removed in `pre tt`. the fine tuning of monospace font is convincing, well done :+1: